flink_scrat/main.py

```diff
@@ -40,6 +40,9 @@ def parse_args():
 
     submit_parser = cmds.add_parser('submit', help="Submit a job to the flink cluster")
 
+    submit_parser.add_argument("--savepoint-path", dest="savepoint_path", required=False,
```

Comment:
If `savepoint-path` and `target-dir` should not be passed togheter, we could make them `mutually exclusive`